packages/core/contracts/polygon/OracleBaseTunnel.sol

```diff
@@ -5,8 +5,7 @@ import "../oracle/interfaces/FinderInterface.sol";
 import "../oracle/implementation/Constants.sol";
 
 /**
- * @notice Enforces lifecyle of price requests for deriving contract. Ensures that price request data is not duplicated
```

Comment:
- The `OracleBaseTunnel` comment [claims](https://github.com/UMAprotocol/protocol/blob/d146d4feddcf99e157a909032a3071e95ca51eb9/packages/core/contracts/polygon/OracleBaseTunnel.sol#L8) that the contract ensures price request data is not duplicated, but the design explicitly allows duplicate price requests and responses to traverse the tunnel. Consider clarifying the comment.